src/Errors.huff

```diff
@@ -0,0 +1,112 @@
+// SPDX-License-Identifier: Unlicence
+
+/// @title HyVM Errors
+
+#define error Error(string)
+
+/* -------------------------------------------------------------------------- */
+/*                               ERROR CONSTANTS                              */
+/* -------------------------------------------------------------------------- */
+
+// "JUMP_FAIL"
+#define constant JUMP_FAIL_ERROR = 0x4a554d505f4641494c0000000000000000000000000000000000000000000000
+#define constant JUMP_FAIL_LENGTH = 0x09
+
+// "NO_SELFDESTRUCT"
+#define constant NO_SELFDESTRUCT_ERROR = 0x4e4f5f53454c4644455354525543540000000000000000000000000000000000
+#define constant NO_SELFDESTRUCT_LENGTH = 0x0F
+
+// "UNUSED"
+#define constant UNUSED_ERROR = 0x554e555345440000000000000000000000000000000000000000000000000000
+#define constant UNUSED_LENGTH = 0x06
+
+// "NOT_OWNER"
+#define constant NOT_OWNER_ERROR = 0x4e4f545f4f574e45520000000000000000000000000000000000000000000000
+#define constant NOT_OWNER_LENGTH = 0x09
+
+// "CHECK_UNAUTHORIZED"
+#define constant CHECK_UNAUTHORIZED_ERROR = 0x434845434b5f554e415554484f52495a45440000000000000000000000000000
+#define constant CHECK_UNAUTHORIZED_LENGTH = 0x0C
+
+// "VERIFICATION_FAILED"
+#define constant VERIFICATION_FAILED_ERROR = 0x564552494649434154494f4e5f4641494c454400000000000000000000000000
+#define constant VERIFICATION_FAILED_LENGTH = 0x0D
+
+/* -------------------------------------------------------------------------- */
+/*                                   ERRORS                                   */
+/* -------------------------------------------------------------------------- */
+
+/// @notice Reverts with an "JUMP_FAIL" message
```

Comment:
Natspec is included in huff ?
Maybe dev for errors ?

Reply:
Yes => [Huff Style Guide](https://docs.huff.sh/style-guide/overview/#natspec-comments)

The use of`@notice` is not as obvious as with Solidity. What is "high-level" ? what is "low-level" ?
I prefer to use `@notice` for what a `.huff` file is exposing and `@dev` for internal macros and technical details.